frontend/express/public/javascripts/countly/countly.template.js

```diff
@@ -1403,7 +1403,6 @@ var AppRouter = Backbone.Router.extend({
             self.addMenuCategory("utilities", {priority: 50});
             self.addMenu("understand", {code: "overview", url: "#/", text: "sidebar.home", icon: '<div class="logo dashboard ion-speedometer"></div>', priority: 10});
             self.addMenu("understand", {code: "analytics", text: "sidebar.analytics", icon: '<div class="logo analytics ion-ios-pulse-strong"></div>', priority: 20});
-            self.addMenu("understand", {code: "engagement", text: "sidebar.engagement", icon: '<div class="logo ion-happy-outline"></div>', priority: 30});
```

Comment:
i dont think you should remove this, since We also add User retention to Engagement menu.
Removing this might not add it.
Although i know Retention will be its own menu according to the new design, but until retention is not transitioned to vue, lets keep this.

Reply:
good catch. I will revert the changes and leave it since retention is still there.

Reply:
sure.